<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreditCalc</class>
 <widget class="QDialog" name="CreditCalc">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>529</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(252, 248, 232);</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>511</width>
     <height>231</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(236, 179, 144);
border-radius:20px;</string>
   </property>
   <widget class="QLabel" name="label_calculation_option">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Вариант расчета</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_loan_amount">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Сумма кредита</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_loan_term">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Срок кредита</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Interest_rate_credit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Процентная ставка</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_type_payments">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>180</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Тип ежемесячных платежей</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_type_credit">
    <property name="geometry">
     <rect>
      <x>275</x>
      <y>10</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Расчет ежемесячного платежа</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>50</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>руб.</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_loan_amount">
    <property name="geometry">
     <rect>
      <x>275</x>
      <y>50</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
	background-color: rgb(252, 248, 232);
	border-radius: 5px;
}
</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>90</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>мес.</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_loan_term">
    <property name="geometry">
     <rect>
      <x>275</x>
      <y>90</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
	background-color: rgb(252, 248, 232);
	border-radius: 5px;
}</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>130</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>%</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_rate_credit">
    <property name="geometry">
     <rect>
      <x>275</x>
      <y>130</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
	background-color: rgb(252, 248, 232);
	border-radius: 5px;
}</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton__type_2">
    <property name="geometry">
     <rect>
      <x>275</x>
      <y>200</y>
      <width>191</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Дифференцированные</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton__type_1">
    <property name="geometry">
     <rect>
      <x>275</x>
      <y>170</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Аннуитетные</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="widget_2" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>310</y>
     <width>511</width>
     <height>131</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(236, 179, 144);
border-radius:20px;</string>
   </property>
   <widget class="QLabel" name="label_month_payment">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Ежемесячный платеж</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_accrued_interest">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Начисленные проценты</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_all">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Общая выплата</string>
    </property>
   </widget>
   <widget class="QLabel" name="month_payment">
    <property name="geometry">
     <rect>
      <x>275</x>
      <y>10</y>
      <width>188</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
	background-color: rgb(252, 248, 232);
	border-radius: 5px;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>10</y>
      <width>30</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>руб.</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>50</y>
      <width>30</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>руб.</string>
    </property>
   </widget>
   <widget class="QLabel" name="accrued_interest">
    <property name="geometry">
     <rect>
      <x>275</x>
      <y>50</y>
      <width>188</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
	background-color: rgb(252, 248, 232);
	border-radius: 5px;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="all">
    <property name="geometry">
     <rect>
      <x>275</x>
      <y>90</y>
      <width>188</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
	background-color: rgb(252, 248, 232);
	border-radius: 5px;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>90</y>
      <width>30</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>руб.</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton_credit_go">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>250</y>
     <width>125</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: rgb(212, 226, 212);
    border-radius: 20px;
    border-color: beige;
}
QPushButton:pressed {
    background-color: rgb(182, 205, 182);
    border-style: inset;
}</string>
   </property>
   <property name="text">
    <string>Рассчитать</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>10</y>
     <width>570</width>
     <height>431</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:20px;
</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_credit_clear">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>250</y>
     <width>125</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: rgb(212, 226, 212);
    border-radius: 20px;
    border-color: beige;
}
QPushButton:pressed {
    background-color: rgb(182, 205, 182);
    border-style: inset;
}</string>
   </property>
   <property name="text">
    <string>Сбросить</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
